fiat-ldap/src/main/java/com/netflix/spinnaker/fiat/config/LdapConfig.java

```diff
@@ -57,5 +57,9 @@ SpringSecurityLdapTemplate springSecurityLdapTemplate() throws Exception {
     String userSearchFilter;
     String groupSearchFilter = "(uniqueMember={0})";
     String groupRoleAttributes = "cn";
+
+    int thresholdToUseGroupFilter = 1000;
+    String groupsFilter = "";
```

Comment:
Can you describe the case where this might be different from `groupSearchFilter`? It seems that we can simply use the same `groupSearchFilter`.

Reply:
`groupSearchFilter` is used by when fiat queries LDAP for each users individually. 
This commit doesn't replace the previous behaviour, but gives a way to make it faster if the user choose to adopt this change.

Reply:
sorry, meant to say `groupSearchBase` here

Reply:
f2f discussion: I see why this may be useful but it seems more apt to be a separate PR. the non-optimised path will have to consider `groupFilter` after applying `groupSearchFilter` if it were introduced.